doc/developers/maintainer.rst

```diff
@@ -119,8 +182,8 @@ Making a release
        $ git checkout master
        $ rm stable
        $ ln -s 0.999 stable
-       $ sed -i "s/latestStable = '.*/latestStable = '0.999';" versionwarning.js
-       $ git commit -m "Update stable to point to 0.999" stable
+       $ sed -i "s/latestStable = '.*/latestStable = '0.999';/g" versionwarning.js
+       $ git commit -am "Update stable to point to 0.999"
```

Comment:
I don't get why you prefer this to explicitly only committing stable

Reply:
the `stable` used to be the only thing changing. Now `versionwarning.js` also changes in this step.

Reply:
Added `git add stable/ versionwarning` on the line above to be explicit.